internal/commands/result.go

```diff
@@ -322,6 +353,13 @@ func convertCxResultsToSarif(results *wrappers.ScanResultsCollection) *wrappers.
 	return sarif
 }
 
+func convertCxResultsToSonar(results *wrappers.ScanResultsCollection) *wrappers.ScanResultsSonar {
+	var sonar = new(wrappers.ScanResultsSonar)
+	sonar.Results = []wrappers.SonarIssues{}
```

Comment:
is this assignment necessary, as the next line assigns to the same variable?

Reply:
It is not, will fix it